### PR TITLE
docs: Added ENTERPRISE Tagging

### PR DESCRIPTION
**Pull request by ashoknayak777**

### CODE REVIEW

File: docs/user-guide/global-configurations/manage-notification.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Manage Notification
 
+## Introduction [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
```
Comment (abhibhaw) [inline docs/user-guide/global-configurations/manage-notification.md]: notification is not enterprise right?

**Review by abhibhaw — CHANGES_REQUESTED** [review]

File: docs/user-guide/global-configurations/manage-notification.md (hunk 1 above)
Comment (ashoknayak777) [reply docs/user-guide/global-configurations/manage-notification.md]: It is Enterprise, I guess...
I have never seen it in any OSS setup so far (attaching snapshot)
image

Comment (ashoknayak777) [reply docs/user-guide/global-configurations/manage-notification.md]: My bad...fixing it...

**Review by abhibhaw — APPROVED** [review]
LGTM

**Review by uxarya-d — APPROVED** [review]

**Pull request merged**